### ISSUE TITLE
Auto select forced subtitles

### ISSUE THREAD

**Issue by devoided** [Stremio/stremio-features#416]
Automatically select forced subtitle track.

Many users don’t use subtitles. But need forced subtitles for moments where the scene introduces a foreign language.

There should be an option for automatically selecting forced subtitle tracks in a language.

**Comment by nathsl** [comment]
👍🏻

**Comment by mpgharris** [comment]
God yes

**Comment by JSS3M3** [comment]
Option to prefer forced subtitles +1. Please give me another reason to stop using Kodi.

**Comment by KvHx** [comment]
Please add this!

**Comment by digitalbarrito** [comment]
+1 to this, or at the very least improvements to the pre-installed official subtitle addons to actually show when a Sub track is Forced. 

Recently watched something that had foreign language sections, and after clicking through every sub labeled "English" I managed to find one that was actually English(forced) but labeled as English.

**Comment by jayseventwo** [comment]
Agree this would be a great idea!

**Comment by josemmc95** [comment]
I think it should be implemented! A basic for me!

**Comment by RayzaEFC** [comment]
Agreed 100% this should be a feature that's added

**Comment by KeKyKo** [comment]
Yes, PLEASE.

**Comment by Alex23pa** [comment]
Yes this is an important feature missing. Most English content does have hard-coded subtitles for the foreign parts.

**Comment by TRtomasz** [comment]
This is impossible to be implemented we have no info which are forced subs

**Comment by devoided** [comment]
Subtitles literally have the word 'forced' in it, so it seems pretty simple
to be able to just match the subtitles with the word 'forced' - it's what I
do manually every time.

**Comment by devoided** [comment]
Subtitles literally have the word 'forced' in it, so it seems pretty simple to be able to just match the subtitles with the word 'forced' - it's what I do manually every time.

**Comment by WhyDaRumGone** [comment]
+1

**Comment by harrydbarnes** [comment]
Surely this can be added in?

**Comment by Arcane25** [comment]
+1 to this. So annoying having to select forced every time... Can't be that hard to make default surely.
